site-scrapers/LowellGeneral/index.js

```diff
@@ -29,20 +26,11 @@ function buildTimeSlotsUrl(stationIdAttr) {
     const locationId = stationIdAttr.split("_")[7];
 
     // start yesterday since they are looking for midnight of a given day
+    const midnight = "+00%3A00%3A00";
     const now = new Date(new Date().getTime() - 86400 * 1000);
-    const startDate =
-        [
-            now.getFullYear(),
-            leadZero(now.getMonth() + 1),
-            leadZero(now.getDate()),
-        ].join("-") + "+00%3A00%3A00";
+    const startDate = moment(now).format("YYYY-MM-DD") + midnight;
     const nowPlus45Days = new Date(now.getTime() + 45 * 86400 * 1000);
```

Comment:
same here - let's use moment and do moment().local().add(45, 'days') (also, `now` is actually yesterday so really this is wrongly 44 days from today)

Reply:
All set